flexget/plugins/modify/convert_magnet.py

```diff
@@ -33,8 +33,10 @@ def magnet_to_torrent(self, magnet_uri, destination_folder, timeout):
         import libtorrent
         params = libtorrent.parse_magnet_uri(magnet_uri)
         session = libtorrent.session()
-        # for some reason the info_hash needs to be bytes but it's a struct called sha1_hash
-        params['info_hash'] = bytes(params['info_hash'])
+        lt_version = [int(v) for v in libtorrent.version.split('.')]
+        if lt_version > [0,16,13,0]:
+            # for some reason the info_hash needs to be bytes but it's a struct called sha1_hash
+            params['info_hash'] = params['info_hash'].to_bytes()
```

Comment:
This is correct. Problem is that the C++ bindings don't support custom data structures, so, it has to be converted to some standard. lt <= 0.16.13 simply used a string to represent the info_hash, and later versions use a custom struct.

Why libtorrent doesn't handle it by itself in the python-libtorrent glue? I have no idea, they should, IMO. Code above is ugly, and this caused breakage on pretty much every project that uses libtorrent.